themes/hugo-tourmaline/static/css/main-site.css

```diff
@@ -951,7 +951,7 @@ ol {
 
 pre, code {
   font-family: "Source Code Pro", monospace;
-  background-color: #f8f8f8;
+  background-color: #f8f8f8 !important;
```

Comment:
What is this needed for?

Reply:
This was the theme fix for those black CSS boxes that appeared around code.